Auth0/SafariAuthenticationSession.swift

```diff
@@ -20,126 +20,40 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import UIKit
-import SafariServices
 #if canImport(AuthenticationServices)
 import AuthenticationServices
 #endif
 
-@available(iOS 11.0, *)
```

Comment:
if 11 is removed here, does that mean we stopped supporting that version? or what would be the class that handles those?

Reply:
It was moved [here](https://github.com/auth0/Auth0.swift/pull/377/files#diff-fc7a91f7e9f7dfde1d6922e21ac64ab7R472).